kroxylicious-integration-test-support/src/main/java/io/kroxylicious/test/tester/KroxyliciousConfigUtils.java

```diff
@@ -126,26 +126,63 @@ public static VirtualClusterListenerBuilder defaultListenerBuilder() {
 
     public static VirtualClusterListenerBuilder defaultPortPerBrokerListenerBuilder(HostPort proxyAddress) {
         return defaultListenerBuilder()
-                .withClusterNetworkAddressConfigProvider(
-                        new ClusterNetworkAddressConfigProviderDefinitionBuilder(PortPerBrokerClusterNetworkAddressConfigProvider.class.getSimpleName())
-                                .withConfig("bootstrapAddress", proxyAddress)
-                                .build());
+                .withNewPortIdentifiesNode()
+                .withBootstrapAddress(proxyAddress)
+                .endPortIdentifiesNode();
```

Comment:
Winning!

So much more readable and clear what is being done.

Reply:
Agreed, it is much nicer.  I squirmed everytime I showed someone a Kroxylicious config.  No more!